src/components/Meme.js

```diff
@@ -81,6 +88,56 @@ export default function Meme() {
 		}
 	}
 
+	
+	//redering the text data in h2 tag 
+	const memeTexts = memeTextData.map(text => {
+		return <h2
+				key={text.id}
+				draggable
+				onDragStart={(e)=>handleDragStart(e, text.id)} 
+				style={text.style}
+				className="meme__text"
+			> 
+				{text.id === "1" ? meme.topText : meme.bottomText} 
```

Comment:
map function also passes the index to the callback, so you can remove the id and instead use that index to work in place of the id.

another reason: we have one feature on adding multiple text, so in that case, we will not be considering this top and bottom text anymore